### ISSUE TITLE
Status of Project Alchemist

### ISSUE THREAD

**Issue by pkhamutou** [jey/alchemist#6]
Hello,

I'd like to ask you about current status of Project Alchemist.

Could you please clarify where the current master is located on Github?

After watching this https://databricks.com/session/alchemist-an-apache-spark-mpi-interface talk, I assumed it's here https://github.com/project-alchemist. There are 4 repositories there and 2 of them are empty  - https://github.com/project-alchemist/AlLib and https://github.com/project-alchemist/ALI.
Is that a working release, or you've just started working on it?

I also found https://github.com/jey/alchemist which a fork of https://github.com/alexgittens/alchemist. Is that a current working release or it was a PoC?
Is it related to https://github.com/project-alchemist?

Best regards,
Pavel Khamutou

**Comment by jey** [comment]
I think https://github.com/project-alchemist/Alchemist is currently the main repo.

 @alexgittens @rustandruin @kai-rothauge: can you comment?

**Comment by alexgittens** [comment]
Hi Pavel,

I have moved away from Alchemist. @kai-rothauge Kai is now managing its development. https://github.com/alexgittens/alchemist is the version that we used to run the reported large-scale experiments on Cori, the NERSC supercomputer; it compiles on Cori and MacBooks. This is no longer being actively developed.

In the interest of making Alchemist more user-friendly (simpler compilation process and support multiple platforms, reduce dependencies, incorporate libraries dynamically, run under containerized systems like Kubernetes), Kai has been rewriting much of the code-base. I believe https://github.com/project-alchemist/Alchemist is the main repo, but I am not sure how usable the code currently is; I recommend leaving a question there if Kai doesn't see this thread.
